src/audienceManager.ts

```diff
@@ -0,0 +1,138 @@
+import { SDKLoggerApi } from './sdkRuntimeModels';
+import {
+    FetchUploader,
+    XHRUploader,
+    AsyncUploader,
+    fetchPayload
+} from './batchUploader';
+import Audience from './audience';
+
+type AudienceResponseMessage = 'ar';
+export interface IAudienceServerResponse {
```

Comment:
We should append the word `Minified` to these interfaces just so we can differentiate from the "Full" interfaces if we ever implement them.